client/layout/masterbar/logged-in.jsx

```diff
@@ -231,13 +230,15 @@ class MasterbarLoggedIn extends React.Component {
 				) }
 				<AsyncLoad require="calypso/my-sites/resume-editing" placeholder={ null } />
 				{ ! domainOnlySite && ! isMigrationInProgress && (
-					<Publish
+					<AsyncLoad
```

Comment:
This is an excellent catch! Since this is conditionally displayed, it doesn't make sense for it to be conditionally loaded 👍